plugins/github_enterprise_server.rb

```diff
@@ -0,0 +1,16 @@
+Plugin.define do
+	name "Github enterprise server"
+	authors [
+	  "Ostorlab",
+	]
+    version "0.1"
+	description "GitHub Enterprise Server is a self-hosted version of the GitHub platform"
+	website "https://docs.github.com/en/enterprise-server@3.14/"
```

Comment:
Do we need to `@3.14` version? it should stay empty so we always get the latest no?

Reply:
the link will always have the latest version, i will see if i can find a link that always redirect to it